_docs/ops/runbook/troubleshooting-logsearch.md

```diff
@@ -123,35 +130,101 @@ mutate {
 ```
 
 Edit the `output` section in `/var/vcap/data/ingestor_syslog/tmp/logstash-restore.conf` and change `index` to:
+
 ```
 index => "logs-%{@index_type}-%{[@metadata][index-date]}"
 ```
 
+### Disable the timecop filter
+
+The default Logsearch-for-CloudFoundry configuration will drop any messages older than 24 hours. When reindexing older data this sanity check needs to be disabled.
+
+To disable the timecop filter set the environment variable `TIMECOP_REJECT_LESS_THAN_HOURS` to match the desired rentention policy:
+
+```sh
+export TIMECOP_REJECT_LESS_THAN_HOURS=$((180 * 24))
+```
+
+### Adjust the IAM role policy to allow the reindexing to read from the S3 bucket
+
+You must go in to the AWS console and adjust the IAM role policy for the environment ingestor you are restoring in, then
+attach the role policy to the ingestor VM you are working in.  When you find the IAM role policy, make sure the Allow Actions match this block:
+
+```
+"s3:PutObject",
+"s3:DeleteObject",
+"s3:GetObject",
+"s3:ListObjects",
+"s3:ListObjectsV2",
+"s3:ListObjectVersions",
+"s3:ListBucket"
+```
+
+Then make sure that Allow Resources matches this block:
+
+```
+"arn:${aws_partition}:s3:::logsearch-*/*",
+"arn:${aws_partition}:s3:::logsearch-*"
+```
+
+Save the role policy adjustments, then attach the role policy to the ingestor VM you are working in.
+
 ### Start the reindexing
 Run logstash passing in your edited configuration file:
 
 ```sh
 export JAVA_HOME=/var/vcap/packages/openjdk-11/jre
-rm /tmp/s3_import.sincedb
+rm /var/vcap/data/ingestor_syslog/tmp/s3_import.sincedb
 /var/vcap/packages/logstash/bin/logstash --path.config /var/vcap/data/ingestor_syslog/tmp/logstash-restore.config
 ```
 
-### Monitor progress
+### Monitor reindexing progress
 
-Logstash will run forever once started. Monitor the progress of the reindex, and stop logstash once the data has been reindexed. Progress can be monitored by tailing the sincedb file which logstash will update after each file it processes.
+Logstash will run forever once started. Monitor the progress of the reindex, and stop logstash once the data has been reindexed. Progress can be monitored by first setting turning the logstash reindexing to a background process and then by tailing the sincedb file which logstash will update after each file it processes:
 
 ```sh
+# While logstash process is running in your foreground:
+<Ctrl>+Z
+bg
+disown
+
```

Comment:
I don't think this is necessary to document - there are several ways to accomplish this that any terminal user should be able to figure out on their own

Reply:
That's true, but in the heat of the moment when dealing with an incident, the last thing I want to do is have to look up things that escape my brain; much easier to have the steps laid out right there!